web/app/templates/tournament_match_form.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <div class="container">
+
+
+    <form action="{{ url_for('.match', tournament_id=match.tournament_id, match_id=match.id) }}" method="post">
+      <input type="hidden" name="_method" value="PUT">
```

Comment:
I was super confused about this until I realized that HTML forms don't allow PUT as a method. Holy crap that sucks that you can't do that!

Maybe leave a comment here to that effect?